specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -0,0 +1,266 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "v1.0-preview",
+    "title": "Form Recognizer Client",
+    "description": "Extracts information from forms and images into structured data based on a model created by a set of representative training forms."
+  },
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/formrecognizer/v1.0-preview",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "Ocr.json#/parameters/Endpoint"
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+
+  "paths": {
+    "/prebuilt/receipt/asyncBatchAnalyze": {
+      "post": {
+        "description": "Batch Read Receipt operation. The response contains a field called 'Operation-Location', which contains the URL that you must use for your 'Get Read Receipt Result' operation.",
+        "operationId": "BatchReadReceipt",
+        "parameters": [
+          {
+            "$ref": "Ocr.json#/parameters/ImageUrl"
+          }
+        ],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header. The 'Operation-Location' URL will expire in 48 hours.",
```

Comment:
This looks like a long running operation

Reply:
It is.  Due to consistency with existing Cognitive Service APIs, we are not able to adopt the latest conventions.  The API review board has agreed with this design.